v3/newLint.sh

```diff
@@ -44,6 +44,7 @@ FILENAME=${LINTNAME:2}
 STRUCTNAME=$3
 
 sed -e "s/PACKAGE/${PATHNAME}/" \
+    -e "s/PASCAL_CASE_SUBST/${STRUCTNAME^}/g" \
```

Comment:
Today I learned that `{$varname^}` will uppercase the first character in the string. Bash is endless.